packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -123,10 +123,20 @@ export interface IGarbageCollectionSnapshotData {
 	deletedNodes: string[] | undefined;
 }
 
+/**
+ * Contains the necessary information to allow DDSes to do incremental summaries
+ */
+export interface IIncrementalContext {
+	sequenceNumber: number;
```

Comment:
Please add comments on the properties. It's not clear what they are and what value should be used for these.